docs/website/generator-errors.md

```diff
@@ -39,6 +39,12 @@ An invalid target framework was passed using the --target-framework argument. Pl
 
 This usually indicates a bug in Xamarin.iOS/Xamarin.Mac; please [file a bug report](https://github.com/xamarin/xamarin-macios/issues/new) with a test case.
 
+### <a name='BI0099'/>BI0099: Internal error *. Please file a bug report with a test case (https://github.com/xamarin/xamarin-macios/issues/new).
+
+This error message is reported when an internal consistency check fails.
+
+This usually indicates a bug in binding generator; please file a new issue on [github](https://github.com/xamarin/xamarin-macios/issues/new) with a test case.
```

Comment:
minor: `a bug in **the** binding generator` (it's a description not a name)
or: `a bug in bgen`